src/libraries/System.Reflection.DispatchProxy/src/System/Reflection/DispatchProxyGenerator.cs

```diff
@@ -831,9 +831,8 @@ public sealed override bool Equals(MethodInfo? left, MethodInfo? right)
                     else if (right == null)
                         return false;
 
-                    // This assembly should work in netstandard1.3,
-                    // so we cannot use MemberInfo.MetadataToken here.
-                    // Therefore, it compares honestly referring ECMA-335 I.8.6.1.6 Signature Matching.
+                    // TODO: switch to use MemberInfo.MetadataToken here.
```

Comment:
Is there an issue tracking this?

Reply:
https://github.com/dotnet/runtime/issues/53285